public/app/plugins/datasource/azuremonitor/components/AzureCheatSheet.tsx

```diff
@@ -0,0 +1,297 @@
+import { css } from '@emotion/css';
+import React, { useEffect, useMemo, useState } from 'react';
+
+import { GrafanaTheme2, SelectableValue } from '@grafana/data';
+import { getBackendSrv, reportInteraction } from '@grafana/runtime';
+import {
+  Button,
+  Card,
+  Collapse,
+  CustomScrollbar,
+  InlineField,
+  Input,
+  LoadingPlaceholder,
+  Select,
+  useStyles2,
+} from '@grafana/ui';
+
+import tokenizer from '../../cloudwatch/language/cloudwatch-logs/syntax';
+import { RawQuery } from '../../prometheus/querybuilder/shared/RawQuery';
```

Comment:
We don't want to be importing from other areas of Grafana core, are these components functions that we can bring into Azure Monitor easily?

Reply:
Yes good catch.